dubbo-admin-frontend/src/components/governance/AccessControl.vue

```diff
@@ -269,7 +267,6 @@ export default {
       ]
     },
     search () {
-      this.filter = document.querySelector('#serviceSearch').value.trim()
```

Comment:
why you remove this?

Reply:
when url like this: ***/governance/access?service=org.apache.dubbo.demo.api.DemoService
mounted () {
    this.setAppHeaders()
    this.setServiceHeaders()
    let query = this.$route.query
    if ('service' in query) {
      this.filter = query['service']  //assign "org.apache.dubbo.demo.api.DemoService" to this.filter
      this.selected = 0
    }
    if ('application' in query) {
      this.filter = query['application']
      this.selected = 1
    }
    this.search()  //document.querySelector('#serviceSearch').value.trim()  will assign an empty string to this.filter
  }

Reply:
sometimes `filter` will get empty value after user input, and  trim() can remove redundant empty spaces,  I think it's better to keep this line